### ISSUE TITLE
Missing info with more than 2 players

### ISSUE THREAD

**Issue by mysterious-imago** [carykh/WordleEdit#5]
When playing a game with 3 players, the bot's DMs with the results of your guess and the eliminated letters do not appear, meaning you have to switch back and forth between the DM and the bot's channel.

**Comment by carykh** [comment]
Oops, that is strange behavior! I'll have a look into it in the next few days. To be honest, I tested WordleEdit with 2 players almost all the time, so I'm not surprised there's bugs when 3+ players are introduced. Anyway, thanks for letting me know!

**Comment by mysterious-imago** [comment]
Absolutely! As soon as I saw the video, I wanted to play around with it. I don't have much coding experience, but I'm happy to report what I see.
